tests/graph/test-data/rust/src/seeding/distributions/ontology/data_type/constraints/string/mod.rs

```diff
@@ -28,13 +28,13 @@ use crate::seeding::distributions::DistributionConfig;
 /// [`StringConstraints`]: type_system::ontology::json_schema::StringConstraints
 #[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
-pub struct StringConstraintsDistributionConfig<'c> {
+pub struct StringConstraintsDistributionConfig {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub min_length: Option<MinLengthDistributionConfig>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub max_length: Option<MaxLengthDistributionConfig>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub pattern: Option<PatternDistributionConfig<'c>>,
+    pub pattern: Option<PatternDistributionConfig>,
```

Comment:
Comment to `PatternDistributionConfig`:

It might be interesting to bridge `arbitrary` and `rand` here, use arbitrary to generate a https://docs.rs/regex-syntax/latest/regex_syntax/ using bytes from rand, then use that in the distribution. That should trivially work and give us a real stress test (`regex_syntax::Ast` provides a `Display` implementation so that it could be trivially turned into a `Regex`).